api/http/handler/webhooks/webhook_execute.go

```diff
@@ -22,7 +22,7 @@ func (handler *Handler) webhookExecute(w http.ResponseWriter, r *http.Request) *
 	}
 
 	webhook, err := handler.WebhookService.WebhookByToken(webhookToken)
-	if err != nil && err != portainer.ErrObjectNotFound {
+	if err != nil {
```

Comment:
When a webhook is not found (as it was deleted or user is fabricating the id), why are we returning 500 instead of 404?